src/nest.ts

```diff
@@ -2,7 +2,9 @@ import {Channel} from 'vega-lite/src/channel';
 
 import {SpecQueryModel} from './model';
 import {SpecQueryModelGroup} from './modelgroup';
-import {SHORT_ENUM_SPEC, EnumSpec, isEnumSpec, stack, stringifyEncodingQueryFieldDef, Query} from './query';
+import {EnumSpec, isEnumSpec, Query, SHORT_ENUM_SPEC} from './query/query';
+import {fieldDef} from './query/shorthand';
```

Comment:
use `fieldDef as fieldDefShorthand` so that you can refer to it in this file as `fieldDefShorthand` for readability